.circleci/build_lib.sh

```diff
@@ -106,8 +106,6 @@ export POLLY_ROOT=`pwd`/toolchains/polly
 ###
 # Clean
 ###
-echo "=====>Cleaning SQLCipher"
```

Comment:
@KhalilBellakrid why was that needed, again?

Reply:
It was needed because we were building this external project before in-source, so when building for another target, it was necessary to clean it, otherwise build would fail, but now I guess we are good to go